api/custom_test.go

```diff
@@ -20,9 +20,9 @@ import (
 	"testing"
 
 	pkgHelm "github.com/banzaicloud/pipeline/pkg/helm"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/api/extensions/v1beta1"
-	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1meta "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I found three different naming of  k8s.io/apimachinery/pkg/apis/meta/v1, it would be fine to use one common name insted of metav1, v1meta and meta_v1